clients/mediaconvert.go

```diff
@@ -27,10 +27,9 @@ var pollDelay = 10 * time.Second
 var retryableHttpClient = newRetryableHttpClient()
 
 const (
-	rateLimitedPollDelay  = 15 * time.Second
-	maxMP4OutDuration     = 2 * time.Minute
-	mp4OutFilePrefix      = "static"
-	maxInputFileSizeBytes = 10 * 1024 * 1024 * 1024 // 10 GiB
+	rateLimitedPollDelay = 15 * time.Second
+	maxMP4OutDuration    = 2 * time.Minute
```

Comment:
Should this be defined elsewhere and not just in clients/mediaconvert.go since we'll use it for livepeer pipeline as well?

Reply:
will do in a follow up PR

Reply:
ok thanks